conf/idp.properties

```diff
@@ -25,8 +25,8 @@ idp.uw.2fa-optout-urlbase = https://groups.uw.edu/group_sws/v3/group/u_weblogin_
 # users soon to be opted in to 2fa
 idp.uw.2fa-optin-soon-urlbase = https://groups.uw.edu/group_sws/v3/group/u_weblogin_optin-2fa-soon/effective_member/
 idp.uw.mfa-soon-cookie = uw-optin-soon-
-# Cookie duration set to 1 week = 7*24*3600 seconds.
-idp.uw.mfa-soon-seconds = 604800
+# Cookie duration set to 5 minutes = 300 seconds.
+idp.uw.mfa-soon-seconds = 300
```

Comment:
Just curious about the change in this duration setting. Probably related to go-live for students, but inquiring minds want to know :)